event_queue.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <stdint.h>
+#include <stdbool.h>
+
+#define EVENT_QUEUE_SIZE 20
+
+typedef struct Event {
+  uint16_t id;
+  uint16_t data;
+} Event;
+
+// Initializes the event queue.
+void event_queue_init(void);
+
+// Raises an event in the global event queue.
+bool event_raise(Event *e);
```

Comment:
As part of our coding standard, we prefix all functions with their module name. It just helps organization better.